### ISSUE TITLE
Metrics tab not working

### ISSUE THREAD

**Issue by louis-she** [aimhubio/aim#1545]
## 🐛 Bug

Click the `Metrics` tab, it shows nothing but a `Something went wrong` notification.

![image](https://user-images.githubusercontent.com/2050207/159153290-93a50ef9-2283-4426-93c7-760bf57594d4.png)

### To reproduce

Use the pytorch_ignite integration. 

### Expected behavior

Show metrics pannel.

### Environment

- Aim Version 3.7.5
- Python version 3.7.10
- pip version 21.1.3
- Ubuntu 20.04
- I'm using the PyTorch Ignite integration.

**Comment by alberttorosyan** [comment]
Hi @louis-she. Are there any error messages in terminal where you ran `aim up` command?

**Comment by louis-she** [comment]
with `--dev` enabled, there is a `404` log.

```
INFO:     172.16.0.70:57002 - "GET /runs/b2b6bc704e1645479c134c16/info HTTP/1.1" 404 Not Found
```

It's wired that why the `Metrics` main tab will access this url, the url looks like to show a specific run

**Comment by roubkar** [comment]
Hey @louis-she , can you please try opening  Aim UI in another browser session (for example  incognito mode) and see if you can still reproduce this issue, seems this is related to URL state caching on UI side

**Comment by louis-she** [comment]
@roubkar i change to safari it is working now. The url goes to `/metrics`. So it is a browser cache issue then.

**Comment by roubkar** [comment]
@louis-she thanks for reporting this issue.
We'll try to reproduce and resolve it asap.

**Comment by roubkar** [comment]
Hey @louis-she, the fix for this issue has been shipped with `v3.8.0`
To upgrade please run `pip install aim --upgrade`

**Comment by louis-she** [comment]
Thanks, I thinks it's good to close the issue
